packages/template-tag-codemod/src/index.ts

```diff
@@ -86,7 +86,12 @@ export function optionsWithDefaults(options?: Options): OptionsWithDefaults {
 type OptionsWithDefaults = Required<Options>;
 
 export async function ensurePrebuild() {
-  if (!existsSync('node_modules/.embroider')) {
+  const stablePrebuildExists = existsSync('node_modules/.embroider/rewritten-app');
```

Comment:
This is not good. It leaves the bug behind to bite us again anytime there's version skew between the embroider that wrote `node_modules/.embroider` and the embroider that's running. We should use a version marker instead.